### ISSUE TITLE
Report parameter name and url value when target url is bad

### ISSUE THREAD

**Issue by westshawn** [grundic/teamcity-web-parameters#13]
web-parameters v 1.6.2
TC server 10.0.2

```
webPopulatedSelect display='prompt' format='json' url='http://a.website/with/bad/or/unreachable/page/' enableEditOnError='true'
```
scheduled trigger causes build to fail to start (expected) with the following error information-
TeamCity build log:
```
Something went wrong during web parameters initialization:
Not Found
```

teamcity-server.log:
```
[2016-11-09 01:23:54,907]  ERROR -   jetbrains.buildServer.SERVER - Error while creating an AgentBuild 
java.lang.RuntimeException: Something went wrong during web parameters initialization:
Not Found
	at ru.mail.teamcity.web.parameters.build.DynamicWebBuildStartContextProcessor.updateParameters(DynamicWebBuildStartContextProcessor.java:54)
	at jetbrains.buildServer.serverSide.impl.RunningBuildImpl.prepareRunners(RunningBuildImpl.java:69)
	at jetbrains.buildServer.serverSide.impl.RunningBuildImpl.createAgentBuild(RunningBuildImpl.java:437)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$1.run(BuildStarter.java:26)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$2$1.run(BuildStarter.java:1)
	at jetbrains.buildServer.util.NamedThreadFactory.executeWithNewThreadName(NamedThreadFactory.java:74)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$2.run(BuildStarter.java:1)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[2016-11-09 01:23:54,907]   WARN -   jetbrains.buildServer.SERVER - Error while starting build {id=521808} 
java.lang.RuntimeException: Something went wrong during web parameters initialization:
Not Found
	at ru.mail.teamcity.web.parameters.build.DynamicWebBuildStartContextProcessor.updateParameters(DynamicWebBuildStartContextProcessor.java:54)
	at jetbrains.buildServer.serverSide.impl.RunningBuildImpl.prepareRunners(RunningBuildImpl.java:69)
	at jetbrains.buildServer.serverSide.impl.RunningBuildImpl.createAgentBuild(RunningBuildImpl.java:437)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$1.run(BuildStarter.java:26)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$2$1.run(BuildStarter.java:1)
	at jetbrains.buildServer.util.NamedThreadFactory.executeWithNewThreadName(NamedThreadFactory.java:74)
	at jetbrains.buildServer.serverSide.impl.BuildStarter$2.run(BuildStarter.java:1)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

For a build with multiple web-select params it would be very helpful to have more information (parameter name and url value) in the build log and the server log.

**Comment by grundic** [comment]
@westshawn, I think it's a great idea! I'll add this functionality soon.

**Comment by grundic** [comment]
Here how it would look like 
<img width="1280" alt="screen shot 2016-11-10 at 22 29 42" src="https://cloud.githubusercontent.com/assets/207278/20195052/3db5c680-a795-11e6-9570-2c93bedd324a.png">

**Comment by grundic** [comment]
I fixed this issue in [v1.6.3](https://github.com/grundic/teamcity-web-parameters/releases/tag/v1.6.3), please have a look.
Thank you!

**Comment by westshawn** [comment]
I verified error text in the build log for scheduled builds includes bad url request info.
Spasibo @grundic !
